### ISSUE TITLE
Failed to compile on aarch64-darwin

### ISSUE THREAD

**Issue by winterqt** [nix-community/nix-eval-jobs#60]
```
[1/2] Compiling C++ object src/nix-eval-jobs.p/nix-eval-jobs.cc.o
[2/2] Linking target src/nix-eval-jobs
FAILED: src/nix-eval-jobs
clang++  -o src/nix-eval-jobs src/nix-eval-jobs.p/nix-eval-jobs.cc.o -Wl,-dead_strip_dylibs -Wl,-headerpad_max_install_names -Wl,-undefined,error /nix/store/7dh9a3rlriwdp8j9120vwwp5s3aqr8yd-nix-2.8.0/lib>
Undefined symbols for architecture arm64:
  "nix::MixEvalArgs::getAutoArgs(nix::EvalState&)", referenced from:
      Proc::Proc(std::__1::function<void (nix::EvalState&, nix::Bindings&, nix::AutoCloseFD&, nix::AutoCloseFD&)> const&)::'lambda'()::operator()() const in nix-eval-jobs.cc.o
  "nix::MixEvalArgs::MixEvalArgs()", referenced from:
      MyArgs::MyArgs() in nix-eval-jobs.cc.o
  "nix::lookupFileArg(nix::EvalState&, std::__1::basic_string_view<char, std::__1::char_traits<char> >)", referenced from:
      releaseExprTopLevelValue(nix::EvalState&, nix::Bindings&) in nix-eval-jobs.cc.o
ld: symbol(s) not found for architecture arm64
```

2e3405f2d10358fd1dde5712b7b61ee7243462d3, NixOS/nixpkgs@3c0f57e36ed0cf9947281e3b31f1bebb7ce5d4a1.

**Comment by winterqt** [comment]
These symbols are definitely exported in the referenced copy of Nix:
```
000000000001619c T __ZN3nix11MixEvalArgs11getAutoArgsERNS_9EvalStateE
0000000000015228 T __ZN3nix11MixEvalArgsC1Ev
0000000000014304 T __ZN3nix11MixEvalArgsC2Ev
0000000000016628 T __ZN3nix13lookupFileArgERNS_9EvalStateENSt3__117basic_string_viewIcNS2_11char_traitsIcEEEE
```

**Comment by winterqt** [comment]
Build just fine in dev shell (with NixOS/nixpkgs@bc41b01dd7a9fdffd32d9b03806798797532a5fe), might be a sandbox issue?

**Comment by adisbladis** [comment]
@winterqt We have the same issue on `x86_64-linux` when building against Nix 2.8.0 which I've fixed in #61 and will tag a release for shortly (once CI has passed).
